backend/fief/services/email/base.py

```diff
@@ -13,6 +13,9 @@ def send_email(
     ):
         ...
 
+    def format_address(self, email: str, name: Optional[str] = None):
```

Comment:
I pilfered this from the Postmark service and made it into a reusable function? I didn't edit the Postmark service to use it, didn't want to overstep.

Reply:
Good call. I've moved that around.

I've also used the function in the postmark service. I think doing so fixed a bug where if `from_name` was `None` then the `To` part would ignore the `to_name`.